imagersite/imager_images/models.py

```diff
@@ -0,0 +1,87 @@
+from __future__ import unicode_literals
+from django.conf import settings
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+from django.utils.translation import ugettext as _
+import uuid
+
+# Create your models here.
+
+@python_2_unicode_compatible
+class Photo(models.Model):
+
+    PRIVATE, SHARED, PUBLIC = 'Pri', 'Shr', 'Pub'
+
+    PUBLISHED_CHOICES = (
+    (PRIVATE, 'Private'),
+    (SHARED, 'Shared'),
+    (PUBLIC, 'Public')
+    )
+
+    def user_directory_path(instance, filename):
+        return 'user_{0}/{1}'.format(instance.user.id, filename)
+
+    photo_id = models.UUIDField(
+               primary_key=True,
+               default=uuid.uuid4,
+               editable=False)
+
+    user = models.OneToOneField(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
+
+    upload = models.ImageField(upload_to='photo_files/%Y-%m-%d')
+    date_created = models.DateTimeField(_('Date Created'))
+    date_modified = models.DateTimeField(_('Date Modified'), auto_now=True)
+    date_uploaded = models.DateTimeField(_('Date Uploaded'), auto_now_add=True)
+    published_status = models.BooleanField(_('Published Status'))
+
+    lat = models.DecimalField(_('Latitude'), max_digits=7, decimal_places=7)
+    lng = models.DecimalField(_('Longitude'), max_digits=7, decimal_places=7)
+
+    camera = models.CharField(_('Camera'), max_length=48, blank=True)
```

Comment:
This might be a good place to use a choice menu instead of just a straight CharField. You might want it such that there's a handful of viable camera models that keep one format and are just selectable. Not a demerit though, just a note.